lbrynet/daemon/Daemon.py

```diff
@@ -246,32 +207,19 @@ def setup(self):
         self.exchange_rate_manager.start()
```

Comment:
Can the exchange manager be a component? Otherwise there is no way to disable it when doing integration tests of the other parts (which don't use the exchange manager).